Sources/String+Prepending.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  String+Prepending.swift
+//  Tests
+//
+//  Created by Igor Ranieri on 23.05.19.
+//
+
+import Foundation
+
+extension String {
+    public mutating func prepend(_ aString: String) {
+        self = aString.appending(self)
+    }
+
+    public func prepending(_ aString: String) -> String {
```

Comment:
wanna add some docs on these?

Reply:
Did not see this comment before. UI playing tricks!